java-client-model/src/main/java/com/cumulocity/model/util/DateTimeUtils.java

```diff
@@ -0,0 +1,164 @@
+package com.cumulocity.model.util;
+
+import com.cumulocity.model.DateTimeConverter;
+import org.joda.time.*;
+import org.joda.time.chrono.ISOChronology;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class DateTimeUtils {
+
+    public static Chronology chronologyUTC() {
+        return ISOChronology.getInstanceUTC();
+    }
+
+    public static Chronology chronologyLocal() {
+        return ISOChronology.getInstance();
+    }
+
+    public static DateTime nowUTC() {
+        return new DateTime(org.joda.time.DateTimeUtils.currentTimeMillis(), chronologyUTC());
+    }
+
+    /**
+     * Don't use {@link DateTimeUtils#nowLocal()} as it adds {@link DateTimeUtils#convertNameToOffset(DateTime)} which breaks
+     * the daylight saving time transition when operating on dates, for example when using minusMonth(1)
+     *
+     * @deprecated use {@link DateTimeUtils#nowDateTimeLocal}
+     */
+    @Deprecated
+    public static DateTime nowLocal() {
```

Comment:
can we get rid of this method now or we have usages? Same for other `@Deprecated` in this class

Reply:
Not many usages, we can definitely switch to `nowDateTimeLocal`
TODO